website/docs/reference/resource-configs/teradata-configs.md

```diff
@@ -0,0 +1,212 @@
+---
+title: "Teradata configurations"
+id: "teradata-configs"
+---
+
+## General
+
+* *Set `quote_columns`* - to prevent a warning, make sure to explicitly set a value for `quote_columns` in your `dbt_project.yml`. See the [doc on quote_columns](https://docs.getdbt.com/reference/resource-configs/quote_columns) for more information.
+
+```yaml
+seeds:
+  +quote_columns: false  #or `true` if you have csv column headers with spaces
+```
+
+* *Enable view column types in docs* - Teradata Vantage has a dbscontrol configuration flag called `DisableQVCI`. This flag instructs the database to create `DBC.ColumnsJQV` with view column type definitions. To enable this functionality you need to:
+  1. Enable QVCI mode in Vantage. Use `dbscontrol` utility and then restart Teradata. Run these commands as a privileged user on a Teradata node:
+      ```bash
+      # option 551 is DisableQVCI. Setting it to false enables QVCI.
+      dbscontrol << EOF
+      M internal 551=false
+      W
+      EOF
+
+      # restart Teradata
+      tpareset -y Enable QVCI
+      ```
+  2. Instruct `dbt` to use `QVCI` mode. Include the following variable in your `dbt_project.yml`:
+      ```yaml
+      vars:
+        use_qvci: true
+      ```
+      For example configuration, see https://github.com/Teradata/dbt-teradata/blob/main/test/catalog/with_qvci/dbt_project.yml.
+
+## Models
+
+### Table
+* `table_kind` - define the table kind. Legal values are `MULTISET` (default for ANSI transaction mode required by `dbt-teradata`) and `SET`, e.g.:
+    ```yaml
+    {{
+      config(
+          materialized="table",
+          table_kind="SET"
+      )
+    }}
+    ```
+  For details, see [CREATE TABLE documentation](https://docs.teradata.com/r/76g1CuvvQlYBjb2WPIuk3g/B6Js16DRQVwPDjgJ8rz7hg).
+* `table_option` - define table options. Legal values are:
```

Comment:
Oh okay! I think rather than providing all the legal values, this section would make more sense listing out the potential input and maybe explaining/linking to the explanations of what they mean

Reply:
Amy, I've not done any usability tests for the docs yet, but my gut feeling is that, if we don't list legal values here, users will be lost when they try to look them up in our docs for [CREATE TABLE](https://docs.teradata.com/r/76g1CuvvQlYBjb2WPIuk3g/B6Js16DRQVwPDjgJ8rz7hg). `CREATE TABLE` docs are generic. They are not dbt-aware. What users need to know is, ho to map options to config keys in dbt. I maintained the same names between dbt and `CREATE TABLE` docs, but the naming convention alone may not be enough. If you are OK with it, my preference is to keep the legal values in dbt docs. As we do usability tests and get more feedback from users, we can tweak.

Reply:
Would you be able to clarify how a user will map those options to the configure keys in the docs? My concern is that folks will not know which legal values map to which field in the configuration. 

It's not that I don't think the legal values are helpful  but more that since a user will not be writing out the create statement in the dbt model, I think it will lead to more confusion on how what they will write in the dbt model file. I want to make sure we are gearing the user to knowing exactly how to create a model file and then filling it out with that they need to materialize their object.

Reply:
Let me know if a call would be easier to discuss this :)

Reply:
@amychen1776 , thanks for the meeting today. I've added a commit that tries to implement what we talked about. Please let me know if this version is easier to follow.